### ISSUE TITLE
Buggy mod

### ISSUE THREAD

**Issue by dkwkwd209** [unilock/DoNotTiltTheCameraInRelationToDamageDirection#2]
When i use it on 1.20.1 servers things like pearl damage doesn't shake the camera directionally but when i take fire damage or fall damage its still the same directional damage...

**Comment by unilock** [comment]
That's strange; ender pearls, fire damage, and fall damage all shake the camera correctly (i.e. ignoring damage) for me, on a Quilt 1.20.1 instance with many other client-side mods installed.

Could you send a complete list of the mods you have installed?

**Comment by dkwkwd209** [comment]
starlight, entity culling, antighost, appleskin, architectury api, bedrodium, boosted brightness,
carpet, carpet fixes, chunky, clean f3, cloth config api, clumps, completeconfig, c2me, dashloader, debugify, dynamic fps, enhanced block entities, fabric language kotlin, faster random, ferritecore, immediately fast, indium, iris shaders, krypton, lithium, log cleaner, logical zoom, memory leak fix, midnightlib, mixin conflict helper, mod menu, model gap fix, more culling, more culling extra, no resource pack warnings, noxesium, player health indicators, reese's sodium options, server pack unlocker, shulker box tooltip, small totem pop, sodium, sodium extra, threadtweak, toomanyplayers, totemcounter, verymanyplayers, yetanotherconfiglib, ukulib, better beds, better fps distance, connectivity, do not tilt camera in relation to damage direction, fastload, get it together drops!, raknetify, servercore, simple armour hud

**Comment by unilock** [comment]
I still can't reproduce the issue, even with that exact set of mods.

<details>
<summary>Mods I tested with</summary>

```
antighost-1.20-fabric0.83.0-1.1.5.jar
appleskin-fabric-mc1.20-2.5.0.jar
architectury-9.1.12-fabric.jar
bedrodium-0.2.2.jar
betterbeds-1.3.0-1.19.3.jar
betterfpsdist-fabric-1.20.1-3.8.jar
boosted-brightness-2.2.0+1.20.1.jar
c2me-fabric-mc1.20.1-0.2.0+alpha.10.96.jar
carpet-fixes-1.20-1.16.1.jar
Chunky-1.3.92.jar
CleanF3-0.4.0-mc1.20.jar
cloth-config-11.1.106-fabric.jar
Clumps-fabric-1.20.1-12.0.0.3.jar
completeconfig-2.5.0.jar
connectivity-fabric-1.20.1-4.5.jar
cupboard-fabric-1.20.1-1.5.jar
dashloader-5.0.0-beta.1+1.20.0.jar
Debugify-1.20.1+2.0.jar
DoNotTiltTheCameraInRelationToDamageDirection-1.0.1+1.19.4.jar
dynamic-fps-2.4.0.jar
enhancedblockentities-0.9+1.20.jar
entityculling-fabric-1.6.2-mc1.20.jar
fabric-api-0.86.1+1.20.1.jar
fabric-carpet-1.20-1.4.112+v230608.jar
fabric-language-kotlin-1.10.8+kotlin.1.9.0.jar
Fastload+1.18.2-1.20-3.4.0.jar
fastrandom-1.3.jar
ferritecore-6.0.0-fabric.jar
getittogetherdrops-fabric-1.20-1.3.1.jar
ImmediatelyFast-1.1.26+1.20.1.jar
indium-1.0.23+mc1.20.1.jar
iris-mc1.20.1-1.6.5.jar
krypton-0.2.3.jar
lithium-fabric-mc1.20.1-0.11.2.jar
logcleaner-1.0.0.jar
logical_zoom-0.0.20.jar
memoryleakfix-fabric-1.17+-1.1.1.jar
midnightlib-fabric-1.4.1.1.jar
MixinConflictHelper-1.2.0.jar
modelfix-1.14-fabric.jar
modmenu-7.2.1.jar
moreculling-1.20.1-0.19.0.jar
MoreCullingExtra-1.1-1.20.1.jar
no-resource-pack-warnings-1.3.0.jar
noxesium-0.1.9.jar
PlayerHealthIndicators-1.1.0-1.20.1.jar
raknetify-fabric-0.1.0+alpha.5.91-all.jar
reeses_sodium_options-1.6.4+mc1.20.1-build.94.jar
servercore-fabric-1.3.7+1.20.1.jar
shulkerboxtooltip-fabric-4.0.4+1.20.1.jar
simple-armor-hud-1.20.1-1.3.1.jar
smallpop-1.20.1-0.2.jar
sodium-extra-0.5.0+mc1.20.1-build.111.jar
sodium-fabric-mc1.20.1-0.5.0.jar
spu-1.6.jar
starlight-1.1.2+fabric.dbc156f.jar
threadtweak-fabric-1.20.1-0.1.0.jar
toomanyplayers-1.20-1.1.4.jar
totemcounter-1.3.0+mc1.20.jar
ukulib-0.7.0+1.20.jar
vmp-fabric-mc1.20.1-0.2.0+beta.7.100-all.jar
yet-another-config-lib-fabric-3.1.0+1.20.jar
```

</details>

Try testing without Carpet + Carpet Fixes, Debugify, and/or Noxesium. It's a mostly random guess, but I think they would be the most likely to modify something like this.

**Comment by unilock** [comment]
Stale.
